### ISSUE TITLE
Error on load "Uncaught TypeError: Illegal constructor"

### ISSUE THREAD

**Issue by rowan-m** [freeCodeCamp/devdocs#932]
Chrome 71.0.3578.21 on a Pixelbook
Tried in incognito mode
```
application-f910f17ac082c07dcc905a99a30c017d8b3290c356d8e2284692ba9528908cf8.js:4068 Uncaught TypeError: Illegal constructor
    at new t (application-f910f17ac082c07dcc905a99a30c017d8b3290c356d8e2284692ba9528908cf8.js:4068)
    at Object.init (application-f910f17ac082c07dcc905a99a30c017d8b3290c356d8e2284692ba9528908cf8.js:3316)
    at HTMLDocument.t (application-f910f17ac082c07dcc905a99a30c017d8b3290c356d8e2284692ba9528908cf8.js:7133)
devdocs.io/:1 Document was loaded from Application Cache with manifest https://devdocs.io/manifest.appcache
devdocs.io/:1 Application Cache Checking event
devdocs.io/:1 Application Cache NoUpdate event
```

**Comment by zzjin** [comment]
same issue here:
```
Uncaught TypeError: Illegal constructor
    at new t (application-87e6ea4fcc6dd3f399f6ba4a05c4b41f8a8f82e25afaa0a978d959e4a3e70fa0.js:4068)
    at Object.init (application-87e6ea4fcc6dd3f399f6ba4a05c4b41f8a8f82e25afaa0a978d959e4a3e70fa0.js:3316)
    at HTMLDocument.t (application-87e6ea4fcc6dd3f399f6ba4a05c4b41f8a8f82e25afaa0a978d959e4a3e70fa0.js:7133)
t @ application-87e6ea4fcc6dd3f399f6ba4a05c4b41f8a8f82e25afaa0a978d959e4a3e70fa0.js:4068
init @ application-87e6ea4fcc6dd3f399f6ba4a05c4b41f8a8f82e25afaa0a978d959e4a3e70fa0.js:3316
t @ application-87e6ea4fcc6dd3f399f6ba4a05c4b41f8a8f82e25afaa0a978d959e4a3e70fa0.js:7133
```
mac os 10.14.2
chrome canary 72
Clean all cache and db do not works.

**Comment by Thibaut** [comment]
This must be a bug in Chrome. Nothing has changed in this code in a long time. Please report the bug to the Chrome team.

**Comment by zzjin** [comment]
Sure that this is a chrome's bug.
chrome have fix it.

**Comment by liamnewmarch** [comment]
This problem occurs when the *Experimental Web Platform features* flag is enabled in Chrome. Tested (and working with the flag disabled) on the following platforms:

Android (Chrome 75.0.3770.143)
macOS (Chrome 75.0.3770.142)

```
Uncaught TypeError: Illegal constructor
    at new t (application-16b8d0312458427caacfdb0ae2f454644e7b306fe6ae95872491122cfb7e7e90.js:4082)
    at Object.init (application-16b8d0312458427caacfdb0ae2f454644e7b306fe6ae95872491122cfb7e7e90.js:3327)
    at HTMLDocument.t (application-16b8d0312458427caacfdb0ae2f454644e7b306fe6ae95872491122cfb7e7e90.js:7181)
```

The specific class which is throwing the error on construction is [CookieStore](https://github.com/freeCodeCamp/devdocs/blob/master/assets/javascripts/lib/cookie_store.coffee) which clashes with a new global built-in type *CookieStore* (an instance of which will be available as `window.cookieStore`) which _is not_ constructible. This is part of the [Cookie Store API](https://chromestatus.com/feature/5658847691669504) is an origin trial.

Given that this is an experimental API this ticket could probably be closed, though renaming the *CookieStore* class or otherwise guarding against the browser built-in would fix the issue.

**Comment by rowan-m** [comment]
Nice discovery, thank you @liamnewmarch DevDocs on planes, trains, and automobiles again!

**Comment by jmerle** [comment]
It's not uncommon for developers to enable experimental features so I'm reopening this to remind myself to rename our CookieStore class when I'm back from vacation.

**Comment by jmerle** [comment]
Fixed in b23b81e693b691e1c5776c72c8a6ca5e547deede by renaming our `CookieStore` to `CookiesStore`.
